tests/functional/aws-node-sdk/test/multipleBackend/objectPutCopyPart/objectPutCopyPartAzure.js

```diff
@@ -675,3 +676,131 @@ function describeF() {
+                    }, err => next(err)),
+                    next => s3.createMultipartUpload(params, (err, res) => {
+                        assert.equal(err, null, 'createMultipartUpload: ' +
+                        `Expected success, got error: ${err}`);
+                        this.currentTest.uploadId = res.UploadId;
+                        next();
+                    }),
+                ], done);
+            });
+
+            it('should copy two 5 MB part from Azure to MPU with Azure ' +
+            'location', function ifF(done) {
+                const params = {
+                    Bucket: azureContainerName,
+                    CopySource:
+                      `${awsBucketName}/` +
+                      `${this.test.keyNameAws}`,
+                    Key: this.test.mpuKeyNameAzure,
+                    PartNumber: 1,
+                    UploadId: this.test.uploadId,
+                };
+                const params2 = {
+                    Bucket: azureContainerName,
+                    CopySource:
+                      `${awsBucketName}/` +
+                      `${this.test.keyNameAws}`,
+                    Key: this.test.mpuKeyNameAzure,
+                    PartNumber: 2,
+                    UploadId: this.test.uploadId,
+                };
+                async.waterfall([
+                    next => s3.uploadPartCopy(params, (err, res) => {
+                        assert.equal(err, null, 'uploadPartCopy: Expected ' +
+                        `success, got error: ${err}`);
+                        assert.strictEqual(res.ETag, `"${fiveMbMD5}"`);
+                        next(err);
+                    }),
+                    next => s3.uploadPartCopy(params2, (err, res) => {
+                        assert.equal(err, null, 'uploadPartCopy: Expected ' +
+                        `success, got error: ${err}`);
+                        assert.strictEqual(res.ETag, `"${fiveMbMD5}"`);
+                        next(err);
+                    }),
+                    next => {
+                        const params = {
+                            Bucket: azureContainerName,
+                            Key: this.test.mpuKeyNameAzure,
+                            MultipartUpload: {
+                                Parts: [
+                                    {
+                                        ETag: `"${fiveMbMD5}"`,
+                                        PartNumber: 1,
+                                    },
+                                    {
+                                        ETag: `"${fiveMbMD5}"`,
+                                        PartNumber: 2,
+                                    },
+                                ],
+                            },
+                            UploadId: this.test.uploadId,
+                        };
+                        s3.completeMultipartUpload(params, (err, res) => {
```

Comment:
for the sake of being thorough, we could get/head the object to make sure the content md5 / content-length is correct